### PR TITLE
add heartbeat endpoint

### PR DESCRIPTION
**Pull request by efagin**
also do a tiny amount of cleanup, like lowercasing schema definition names and adding an "id" stage variable for the heartbeat's response.

### CODE REVIEW

**Comment by efagin** [comment]
@e0d mind reviewing this PR? not urgent - could use your feedback on the value of this new endpoint (/heartbeat). Note that the heartbeat response contains a fixed (for now) version 1.0.0 and an ID based on a stage variable if set. I updated my deploy script to plug the git commit hash in as the ID value so we could easily figure out the Swagger index file that was used to build a live gateway. And, look ma, no back-end (it's a "mock" endpoint that just returns a string - other open edx installs may wish to change this to be an actual backend or get rid of it entirely).

File: swagger/heartbeat.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+definitions:
+  heartbeat:
+    required:
+      - version
+    properties:
+      version:
+        type: "string" 
+      id:
+        type: "string"
```
Comment (clintonb) [inline swagger/heartbeat.yaml]: Nit: add a newline

File: swagger/oauth.yaml [hunk 2]
```diff
@@ -98,4 +98,4 @@ definitions:
         type: "string"
       expires_in:
         type: "integer"
-        format: "int64"
+        format: "int64"
```
Comment (clintonb) [inline swagger/oauth.yaml]: Nit: add a newline

**Comment by clintonb** [comment]
:+1:

**Comment by efagin** [comment]
Thanks @clintonb - newlines addressed. My bad. I'm going to merge to master now, but I'd still like Ed to take a look at some point to determine if the heartbeat approach is reasonable.

**Pull request merged**